retrieve-digests/pkg/retrieve_digests.go

```diff
@@ -0,0 +1,49 @@
+package pkg
+
+import (
+	"context"
+	"log"
+	"strings"
+)
+
+// RunRetrieveDigests contains the core logic for processing repositories
+func RunRetrieveDigests(project string) (int, int, error) {
+	projectURI := strings.ToLower(project)
+	projectURI = strings.ReplaceAll(projectURI, ":", "/")
```

Comment:
```suggestion
	projectURI := strings.ReplaceAll(strings.ToLower(project), ":", "/")
```
In order to remove 1 not needed line